App crashes at startup on release builds

```
  init {
    Timber.plant(if (isReleaseBuild()) crashReportingTree else Timber.DebugTree())
  }
```

raises the following exception

```
16:26:54.418  E  FATAL EXCEPTION: main
                 Process: com.google.android.ground.openforis.release, PID: 2664
                 java.lang.RuntimeException: Unable to instantiate application com.google.android.ground.GroundApplication package com.google.android.ground.openforis.release: kotlin.UninitializedPropertyAccessException: lateinit property crashReportingTree has not been initialized
                 	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1553)
                 	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1463)
                 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7532)
                 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
                 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2419)
                 	at android.os.Handler.dispatchMessage(Handler.java:106)
                 	at android.os.Looper.loopOnce(Looper.java:257)
                 	at android.os.Looper.loop(Looper.java:368)
                 	at android.app.ActivityThread.main(ActivityThread.java:8832)
                 	at java.lang.reflect.Method.invoke(Native Method)
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:572)
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1049)
                 Caused by: kotlin.UninitializedPropertyAccessException: lateinit property crashReportingTree has not been initialized
                 	at com.google.android.ground.GroundApplication.getCrashReportingTree(GroundApplication.kt:34)
                 	at com.google.android.ground.GroundApplication.<init>(GroundApplication.kt:38)
                 	at java.lang.Class.newInstance(Native Method)
                 	at android.app.AppComponentFactory.instantiateApplication(AppComponentFactory.java:76)
                 	at androidx.core.app.CoreComponentFactory.instantiateApplication(CoreComponentFactory.java:52)
                 	at android.app.Instrumentation.newApplication(Instrumentation.java:1301)
                 	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1545)
                 	at android.app.LoadedApk.makeApplicationInner(LoadedApk.java:1463) 
                 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7532) 
                 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0) 
                 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2419) 
                 	at android.os.Handler.dispatchMessage(Handler.java:106) 
                 	at android.os.Looper.loopOnce(Looper.java:257) 
                 	at android.os.Looper.loop(Looper.java:368) 
                 	at android.app.ActivityThread.main(ActivityThread.java:8832) 
                 	at java.lang.reflect.Method.invoke(Native Method) 
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:572) 
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1049) 
16:26:54.422  E  uah_event_acquire ! cmdId: 0 ,strPkg: com.google.android.ground.openforis.release, strIdent: com.android.server.wm, timeout: 600
```

--- a/ground/src/main/java/com/google/android/ground/GroundApplication.kt
+++ b/ground/src/main/java/com/google/android/ground/GroundApplication.kt
@@ -34,14 +34,11 @@ class GroundApplication : MultiDexApplication(), Configuration.Provider {
   @Inject lateinit var crashReportingTree: CrashReportingTree
   @Inject lateinit var workerFactory: HiltWorkerFactory
 
-  init {
-    Timber.plant(if (isReleaseBuild()) crashReportingTree else Timber.DebugTree())
-  }
-
   private fun isReleaseBuild(): Boolean = BuildConfig.BUILD_TYPE.contentEquals("release")
 
   override fun onCreate() {
     super.onCreate()
+    Timber.plant(if (isReleaseBuild()) crashReportingTree else Timber.DebugTree())
     if (!isReleaseBuild()) {
       Timber.d("DEBUG build config active; enabling debug tooling")
 

